ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -235,23 +182,54 @@ then
 
 		if [ $FAILSAFE_AUX != none ]
 		then
-			pwm failsafe -d ${OUTPUT_AUX_DEV} ${FAILSAFE}
+			pwm failsafe -p ${FAILSAFE} -d ${OUTPUT_AUX_DEV}
```

Comment:
Don't we need the `-c ${PWM_AUX_OUT}` as well?

Reply:
I wasn't quite sure if I should add those before I heard back from @knopancakes on his testing, but I agree, it seems it should go in.  I'll push that up momentarily.

Thanks!

Reply:
Changes pushed up.  I was a little unsure if I should continue the use of `PWM_ACHDIS` instead of `PWM_AUX_OUT` which I opted for at line 185.  This was in anticipation of deprecating `PWM_ACHDIS`.  Let me know if I should handle this differently.  Thanks!

Reply:
`PWM_AUX_OUT` is fine, and we can then remove `PWM_ACHDIS`.